completion/available/github-cli.completion.bash

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+cite "about-completion"
+about-completion "GitHub CLI completion"
+
+if command -v which gh >/dev/null 2>&1; then
+  if command -v which brew >/dev/null 2>&1; then
```

Comment:
Should we really be checking for a Homebrew installation here? This is pretty much what the `system` completion does: https://github.com/Bash-it/bash-it/blob/master/completion/available/system.completion.bash - I'd rather use that than checking for `brew` in every single completion file.

Reply:
Sweet! I didn't even realize that was there.